src/frontend/state/GlobalState.tsx

```diff
@@ -421,7 +421,7 @@ class GlobalState extends PureComponent<Props> {
     }
 
     let updates = this.state.gameUpdates
-    if (checkUpdates && library) {
+    if (checkUpdates) {
```

Comment:
maybe we can just remove these checks and then call the checkGameUpdate on every refresh for runner 'all' or 'legendary'

Reply:
GOG Games can also have updates

Other than that, I believe this system is currently working fine. Why change it?